lte/gateway/python/magma/mobilityd/dhcp_client.py

```diff
@@ -272,7 +272,7 @@ def _process_dhcp_pkt(self, packet, state: DHCPState):
                         ip_offered + "/" + "32", strict=False,
                     )
 
-                dhcp_server_ip = None
+                dhcp_server_ip: str
```

Comment:
It's not always set so it should be an optional string.
```suggestion
                dhcp_server_ip: Optional[str] = None
```